internal/bundle/plain_test.go

```diff
@@ -0,0 +1,96 @@
+package bundle
+
+import (
+	"io/fs"
+	"testing/fstest"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Plain bundle v0 loader", func() {
```

Comment:
@joelanford @timflannagan Do we have a preference for using Ginkgo vs Go's standard testing framework when unit testing? We use Go's standard framework for some instance and Ginkgo for others .

Reply:
I personally prefer ginkgo, but I realize its a contentious debate. I'm more interested in consistency and ease of adding new test cases to existing test suites than I am about any particular framework.